users/views.py

```diff
@@ -69,8 +76,6 @@ def activate(request, uidb64, token):
         user.save()
         login(request, user)
         # return redirect('home')
-        return HttpResponse(
-            "Thank you for your email confirmation. Now you can login your account."
-        )
+        return HttpResponseRedirect(reverse_lazy("confirmation_complete"))
```

Comment:
Same here, I think `reverse` should be enough.

Reply:
Yes, it's true, reverse is enough, changing it now.